bin/resource_management/update_resource_format.py

```diff
@@ -0,0 +1,31 @@
+import sys

```

Comment:
this is a script so mark it executable and add a `#!/usr/bin/env python3` line to the top